src/utils/BookmarkIcon.js

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+
+export default function BookmarkIcon(props) {
```

Comment:
Icons should be kept in a separate folder(../assets/images/icons). You would need to create that.
Also, assets should be out of `src` folder.

Reply:
moved to assests